teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<ul><li>Could you add a reference were the reader can find more information on the descriptors you exemplified in the first paragraph? ' As done for molecules, the proteins ....embeddings).</li><li>'when the whole protein wants to be incorporated to the model, a multiple sequence alignment can be performed. The final descriptor will have as many features as the number of features per amino acid multiplied by the number of aligned positions. To take into account, gaps in the alignment will receive zeroes in the descriptor.' -&gt; 'If the entire protein is to be included in the model, a multiple sequence alignment can be performed. The final descriptor has as many entries as the number of features per amino acid multiplied by the number of aligned positions. To account for gaps in the alignment, zeros are introduced in the descriptor.'</li><li>Add link to ProDec</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>